### DISCUSSION TITLE
Welcome to rubick Discussions!

### DISCUSSION THREAD

**Discussion by muwoo** [rubickCenter/rubick#29] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by radiorz** [comment]
在哪里提新需求呢?!
Where to commit new feature need?

**Reply by muwoo** [reply]
新开个issue就好了。

**Comment by firstmeet1108** [comment]
没有交流群之类的吗
